cached_proc_macro/src/cached.rs

```diff
@@ -27,7 +36,7 @@ struct MacroArgs {
     #[darling(default)]
     option: bool,
     #[darling(default)]
-    sync_writes: bool,
+    sync_writes: SyncWriteMode,
```

Comment:
Cannot we have `Option<SyncWriteMode>`
And define this
```
enum SyncWriteMode {
    #[default]
    Default,
    ByKey,
}
```

Reply:
We could, but would that make a difference?

Reply:
Then I think we can have these cases:
`#[cached()]`, `#[cached(sync_writes)]` and `#[cached(sync_writes = "by_keys")]`

Reply:
oh yeah I didn't know about this case `#[cached(sync_writes)]` never used that :+1: